webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -126,26 +125,25 @@ class Test_generate_create_bug_url:
         + "comment=%(description)s&"
         + "format=__default__"
     )
+    CRASH_ID = "70dda764-a402-4ca3-b806-c38dd0240328"
 
-    @staticmethod
-    def _create_report(**overrides):
+    def _create_report(self, **overrides):
         default = {
             "signature": "$&#;deadbeef",
-            "uuid": "00000000-0000-0000-0000-000000000000",
+            "uuid": self.CRASH_ID,
             "cpu_arch": "x86",
             "os_name": None,
         }
         return dict(default, **overrides)
 
-    @staticmethod
-    def _extract_query_string(url):
```

Comment:
There's no need for these to be static methods and when they are static methods, they can't access instance properties/methods. So I changed them.